gc_tests/run_tests.rs

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) 2019 King's College London created by the Software Development
+// Team <http://soft-dev.org/>
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0>, or the MIT license <LICENSE-MIT
+// or http://opensource.org/licenses/MIT>, or the UPL-1.0 license
+// <http://opensource.org/licenses/UPL> at your option. This file may not be
+// copied, modified, or distributed except according to those terms.
+
+use std::{env, path::PathBuf, process::Command};
+
+use lang_tester::LangTester;
+use tempdir::TempDir;
+
+static CRATE_NAME: &'static str = "gcmalloc";
+
+fn main() {
+    let tempdir = TempDir::new("gc_tester").unwrap();
+    LangTester::new()
+        .test_dir("gc_tests/tests")
+        .test_file_filter(|p| p.extension().unwrap().to_str().unwrap() == "rs")
+        .test_extract(|s| {
+            Some(
+                s.lines()
+                    .skip_while(|l| !l.starts_with("//"))
+                    .take_while(|l| l.starts_with("//"))
```

Comment:
I think you only need the `skip_while` or the `take_while`, but not both (since they're inverses of each other).